Formula/oclgrind.rb

```diff
@@ -21,40 +21,51 @@ class Oclgrind < Formula
 
   depends_on "cmake" => :build
   depends_on "llvm@13"
+  depends_on "readline"
```

Comment:
A bit odd to suddenly need this on macOS too.

Reply:
Existing bottles are indirectly linked via `llvm@13 --> python@3.10 --> readline`. Since I am looking into removing Python dependency from LLVM (at least in LLVM 14+), it is better to be explicit on these.

https://github.com/jrprice/Oclgrind/blob/v21.10/CMakeLists.txt#L164-L166

Reply:
We could just not build LLDB on LLVM 13. The versioned LLVM formulae should just provide the minimum required by dependents. 

Though admittedly in the case of formulae like `zig` is a lot of it. I don't think LLDB is needed, though -- typically just `libLLVM`, and occasionally `libclang` and `lld`. We can almost certainly cut `openmp`, maybe `clang-tools-extra`.